src/content/docs/tutorial-create-alerts/improve-with-alerts.mdx

```diff
@@ -0,0 +1,94 @@
+1. Are the alerts telling us something about a resource that needs to be fixed? If so, then fix the problem and see if the alert volume decreases.
+2. Are the alerts telling us about something that actually requires an immediate response? If not, then adjust or disable the policy.
+3. Are the policy thresholds set properly? If not, then consider adjusting the thresholds.
+
+Your response teams should deal with the incident alerts following these guidelines during and after the AQM orientation:
+
+* If you look at an alert and decide to take any sort of further investigative action, acknowledge the alert.
+* If you typically close an alert without doing anything else, do not acknowledge the alert.
+* If the incident alert is always on, do not close or acknowledge it. 
+    </Collapser>
+    <Collapser
+        id="gather-data"
+        title="4. Gather your data"
+    >
+It takes some time to accumulate your alert data from the dashboard. You should wait at least 2 weeks to gather this data, and regulary check to ensure that the incident responders for your alerts are following the guidelines outlined in the previous step.
+    </Collapser>
+    <Collapser
+        id="second-enablement"
+        title="5. Perform the second AQM orientation"
+    >
+After two weeks, you should have enough data to analyze and begin your alert improvement process. To improve your system using the alert data, follow the steps below:
+1. Analyze the week-over-week trends in your KPIs. Find the areas that need to be fixed and you can begin finding ways to improve them.
+2. Use the data to map the current state of alert quality, identifying areas where improvement has positively impacted the business and areas where problems have resulted in negative outcomes.
+3. Use the dashboard to identify the noisiest incident policies. 
+4. Review the policies identified in the previous step. For each policy, try to determine if the alert is relevant, properly configured, and what the alert tells you about problems that may need to be addressed.
+5. Identify what areas you can work on to improve the policies you reviewed. This should be a technical analysis, and should end with recommendations in how to fix problems in your system that trigger the alert, how to tune policies that need improvement, or how to fix any gaps in your instrumentation
+    </Collapser>
+</CollapserGroup>
+
+After completing the procedure above, you're well on your way to using your alerts to improve your system and provide a positive impact to your organiation. This is only the beginning though: there's a lot more to alert quality management than what we've covered here. For more detailed information on AQM, as well as KPIs and the alert improvement process, see our [AQM docs](/docs/new-relic-solutions/observability-maturity/uptime-performance-reliability/alert-quality-management-guide/#perform-enablement-one).
```

Comment:
I think we need an outro for the whole series

Reply:
It feels odd to still have an AQM OMA doc as well as this tutorial

Reply:
There is a lot more to AQM than what I have here. When I was reviewing the docs this was the section that stood out to me as the most direct, actionable, and likely to grab a reader, so I took it and repurposed it. We can discuss in our meeting.